### PR TITLE
HAWQ-897. Add feature test for create table distribution with new test framework

### PR DESCRIPTION
**Pull request by jiny2**

### CODE REVIEW

**Comment by linwen** [comment]
+1

**Comment by yaoj2** [comment]
LGTM

File: src/test/feature/catalog/test_create_table.cpp [hunk 1]
```diff
@@ -22,26 +22,26 @@ class TestCreateTable : public ::testing::Test {
 TEST_F(TestCreateTable, TestCreateTable1) {
   hawq::test::SQLUtility util;
   // prepare
-  util.execute("DROP TABLE IF EXISTS aggtest");
```
Comment (hsyuan) [inline src/test/feature/catalog/test_create_table.cpp]: The easier way to do this is creating a new schema `X`, set the default search_path to `X`. At the end of the test, drop the schema. In this way, you don't need to take care of dropping tables when you add new test cases. This is how GPDB is doing.

Comment (paul-guo-) [inline src/test/feature/catalog/test_create_table.cpp]: I think jinyi's intent is to test "create/drop table".

Actually the constructor and destructor of class hawq::test::SQLUtility are exactly doing what you've suggested.

See src/test/feature/lib/sql_util.cpp for details.

**Pull request closed without merging**